### ISSUE TITLE
-Wenum-conversion in drivers/gpu/drm/amd/amdgpu/gfx_v9_4.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#849]
```
../drivers/gpu/drm/amd/amdgpu/gfx_v9_4.c:967:35: warning: implicit conversion from enumeration type 'enum amdgpu_ras_block' to different enumeration type 'enum ta_ras_block' [-Wenum-conversion]
        block_info.block_id = info->head.block;
                            ~ ~~~~~~~~~~~^~~~~
1 warning generated.
```

Patch submitted: https://lore.kernel.org/lkml/20200130012435.49822-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Accepted: https://cgit.freedesktop.org/~agd5f/linux/commit/?id=264ead17d0d245618fc3906e1980950a1718b80f

**Comment by nathanchance** [comment]
Fixed in next-20200203: https://git.kernel.org/next/linux-next/c/968162204adf5f2292214356978357e9dcd35de1
